receiver/statsdreceiver/protocol/statsd_parser.go

```diff
@@ -31,16 +31,18 @@ var (
 )
 
 func getSupportedTypes() []string {
-	return []string{"c", "g"}
+	return []string{"c", "g", "h", "ms"}
 }
 
 const TagMetricType = "metric_type"
 
 // StatsDParser supports the Parse method for parsing StatsD messages with Tags.
 type StatsDParser struct {
-	gauges           map[statsDMetricdescription]pdata.InstrumentationLibraryMetrics
-	counters         map[statsDMetricdescription]pdata.InstrumentationLibraryMetrics
-	enableMetricType bool
+	gauges             map[statsDMetricdescription]pdata.InstrumentationLibraryMetrics
+	counters           map[statsDMetricdescription]pdata.InstrumentationLibraryMetrics
+	timers             []pdata.InstrumentationLibraryMetrics
```

Comment:
maybe `timersAndDistributions` to be consistent with statsd exporter terminology. I can't tell whether "timer" or "timing" is more correct, but would probably accept both as well as "ms" and "h" in the `statsd_type` setting proposed above.